### PR TITLE
Feat(CX-3367): redesign the swa landing page part 3

### PR DESCRIPTION
**Pull request by dariakoko**
The type of this PR is: **Feat**

This PR solves [CX-3367]

### Description

https://user-images.githubusercontent.com/36167539/217576322-2648e4e5-bf85-4593-8d8a-9a73647a02c2.mov

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CX-3367]: https://artsyproduct.atlassian.net/browse/CX-3367?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/MeetTheSpecialists.tsx [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+  const { trackEvent } = useTracking()
+
+  const [selectedSpecialty, setSelectedSpecialty] = useState<Specialty>(
+    "auctions"
+  )
+  const [specialistsTooDisplay, setSpecialistsTooDisplay] = useState(
+    SPECIALISTS.filter(i => i.specialty === "auctions")
+  )
+
+  const contactTheSpecialistClick = () => {
```
Comment (lordkiz) [inline src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/MeetTheSpecialists.tsx:54]:
```suggestion
  const trackContactTheSpecialistClick = () => {
```

File: src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/MeetTheSpecialists.tsx [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+                SPECIALISTS.filter(i => i.specialty === pill.type)
+              )
+            }}
+          >
+            {pill.title}
+          </Pill>
+        ))}
+      </Flex>
+      <Rail
+        title=""
```
Comment (olerichter00) [inline src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/MeetTheSpecialists.tsx:104]: **question:** Is this empty title needed?

File: src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/SpecialistsData.tsx [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+  "https://files.artsy.net/images/08_CVP_About_Follow (2).png",
+  imageOptions
+)
+
+const TEST_imagePriveteSalesAndAdvisory = cropped(
+  "https://files.artsy.net/images/06_CVP_About_Sell.png",
+  imageOptions
+)
+
+const TEST_CollectorServices = cropped(
```
Comment (olerichter00) [inline src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/SpecialistsData.tsx:33]: **question:** Is the `TEST_` prefix needed?

**Review by olerichter00 — APPROVED** [review]
Looking good 🌟

**Review by MounirDhahri — DISMISSED** [review]
Niiice

File: src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/SpecialistsData.tsx (hunk 3 above)
Comment (dariakoko) [reply src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/SpecialistsData.tsx:33]: I'm using it to point to the fact that these images are not final and should be replaced when we have actual photos of specialists

File: src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/MeetTheSpecialists.tsx (hunk 2 above)
Comment (dariakoko) [reply src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/MeetTheSpecialists.tsx:104]: I think so, it's a required prop for the Rail component

File: src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/MeetTheSpecialists.tsx (hunk 1 above)
Comment (dariakoko) [reply src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/MeetTheSpecialists.tsx:54]: Noted, thanks!

**Review by olerichter00 — APPROVED** [review]

**Pull request merged**